.github/BOTMETA.yml

```diff
@@ -1036,7 +1036,7 @@ files:
   $modules/read_csv.py:
     maintainers: dagwieers
   $modules/redfish_:
-    ignore: jose-delarosa
+    ignore: jose-delarosa TSKushal
```

Comment:
I'm still not sure why this is needed; I'm not entirely sure what it's supposed to control with the CI workflow. @felixfontein, can you explain what the ignore statement does here? I wasn't able to find much in the bot's documentation.

Reply:
Every person mentioned in the `author` field of a module (or plugin) needs to show up in BOTMETA.yml as well, either in `maintainers` or in `ignore`. Usually adding yourself on the `authors` list means that you should at least for some time also serve as a maintainer for that module, so directly adding yourself to `ignore` is not a good idea.

Reply:
About what `ignore` does: it basically says "do not ping this person when there is a PR/issue that affects this module".

Reply:
Got it; that makes sense why Jose is there since he's the original author, but has moved on from that job.

Reply:
Hope this is clarified now. Can I resolve it?

Reply:
@TSKushal given that you're adding yourself as an author, you should instead add yourself as a maintainer instead of adding yourself to ignore immediately.

Reply:
Done. Looking forward to working with everyone 😄